app/V1Module/presenters/AssignmentsPresenter.php

```diff
@@ -193,10 +193,15 @@ public function actionUpdateDetail(string $id) {
     }
 
     if ($this->isRequestJson()) {
-      $disabledRuntimeIds = $req->getPost("disabledRuntimeEnvironments");
-      $disabledRuntimes = array_map([$this->runtimeEnvironments, "findOrThrow"], $disabledRuntimeIds);
+      $disabledRuntimeIds = $req->getPost("disabledRuntimeEnvironmentIds");
+      if ($disabledRuntimeIds !== null) {
+        if (!is_array($disabledRuntimeIds)) {
```

Comment:
Instead of this you can use validation check in `@Param` annotation

Reply:
I'm not sure which one do you mean. Null or array validation?

Reply:
I'm refering to the line with array check... so I would say array validation

Reply:
Yeah, that could be done... I wasn't sure how validation works with optional parameters